schema/install_schema.go

```diff
@@ -7,65 +7,65 @@ import (
+	GrubOptionsSchema   `json:"grub_options,omitempty" yaml:"grub_options,omitempty"`
+	Image               string `json:"image,omitempty" description:"Use a different container image for the installation" yaml:"image,omitempty"`
 	PowerManagement
-	SkipEncryptCopyPlugins bool                `json:"skip_copy_kcrypt_plugin,omitempty"`
-	Partitions             ElementalPartitions `json:"partitions,omitempty" mapstructure:"partitions"`
-	GrubDefEntry           string              `json:"grub-entry-name,omitempty" mapstructure:"grub-entry-name"`
-	ExtraPartitions        []*Partition        `json:"extra-partitions,omitempty" mapstructure:"extra-partitions"`
-	Force                  bool                `json:"force,omitempty" mapstructure:"force"`
-	ExtraDirsRootfs        []string            `json:"extra-dirs-rootfs,omitempty" mapstructure:"extra-dirs-rootfs"`
-	Active                 Image               `json:"system,omitempty" mapstructure:"system"`
-	Recovery               Image               `json:"recovery-system,omitempty" mapstructure:"recovery-system"`
-	Passive                Image               `json:"passive,omitempty" mapstructure:"recovery-system"`
+	SkipEncryptCopyPlugins bool                `json:"skip_copy_kcrypt_plugin,omitempty" yaml:"skip_copy_kcrypt_plugin,omitempty"`
+	Partitions             ElementalPartitions `json:"partitions,omitempty" mapstructure:"partitions" yaml:"partitions,omitempty"`
+	GrubDefEntry           string              `json:"grub-entry-name,omitempty" mapstructure:"grub-entry-name" yaml:"grub-entry-name,omitempty"`
+	ExtraPartitions        []*Partition        `json:"extra-partitions,omitempty" mapstructure:"extra-partitions" yaml:"extra-partitions,omitempty"`
+	Force                  bool                `json:"force,omitempty" mapstructure:"force" yaml:"force,omitempty"`
+	ExtraDirsRootfs        []string            `json:"extra-dirs-rootfs,omitempty" yaml:"extra-dirs-rootfs,omitempty" mapstructure:"extra-dirs-rootfs"`
+	Active                 Image               `json:"system,omitempty" mapstructure:"system" yaml:"system,omitempty"`
+	Recovery               Image               `json:"recovery-system,omitempty" mapstructure:"recovery-system" yaml:"recovery-system,omitempty"`
+	Passive                Image               `json:"passive,omitempty" mapstructure:"recovery-system" yaml:"passive,omitempty"`
 }
 
 type Image struct {
-	Size       uint   `json:"size,omitempty" mapstructure:"size"`
-	Source     string `json:"uri,omitempty" mapstructure:"uri"`
+	Size   uint   `json:"size,omitempty" mapstructure:"size" yaml:"size,omitempty"`
+	Source string `json:"uri,omitempty" mapstructure:"uri" yaml:"uri,omitempty"`
 }
 
 type Partition struct {
-	Name string `json:"name,omitempty"`
-	Size uint   `json:"size,omitempty" mapstructure:"size"`
-	FS   string `json:"fs,omitempty" mapstrcuture:"fs"`
+	Name string `json:"name,omitempty" yaml:"name,omitempty"`
```

Comment:
I think name is ignored in the install struct

Reply:
@nianyush Same goes w/ Name, it's not in the Install struct. Please advise.